tailwind.config.js

```diff
@@ -286,9 +286,13 @@ module.exports = {
       gap: theme => theme('spacing'),
 
       borderRadius: {
-        0.75: '0.1875rem', // 3px
-        1.25: '0.3125rem', // 5px
         2.5: '0.625rem', // 10px
+        3: 3,
+        5: 5,
+        6: 6,
+        7: 7,
+        8: 8,
+        10: 10,
```

Comment:
Please do something with these classes names because it may be not obvious that `rounded-3` gives smaller border radius than `rounded-2.5`